### PR TITLE
Record bergamot translator version

### PR DESCRIPTION
**Pull request by eu9ene**
- added extraction and recording of bergamot-translator version
- moved telemetry schema generation to `scripts` folder

### CODE REVIEW

File: extension/manifest.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
             "controller/translation/Translation.js",
             "controller/translation/TranslationMessage.js",
             "controller/translation/translationWorker.js",
+            "controller/translation/bergamotTranslatorVersion.js",
```
Comment (andrenatal) [inline extension/manifest.json:39]: Is there a tab here? Just remove it please

**Review by andrenatal — APPROVED** [review]
Cam we have these instructions in the readme file?

File: extension/manifest.json (hunk 1 above)
Comment (eu9ene) [reply extension/manifest.json]: done

**Comment by eu9ene** [comment]
> Cam we have these instructions in the readme file?

added instructions

**Review by andrenatal — APPROVED** [review]

**Pull request merged**